### ISSUE TITLE
Reflow When Table is Horizontally Scrolled

### ISSUE THREAD

**Issue by kev22257** [mkoryak/floatThead#300]
Windows 7 (Version 6.1.7601)

Firefox 45.0.1
IE 11
Chrome 49.0.2623.110 m

jQuery.floatThead 1.4.0
jQuery v2.1.4

When a table is inside a div that has horizontal scroll and the user has scrolled some amount to the right a reflow on the table will cause the headers to be misaligned with the table contents.  Scrolling any amount in any direction will correct the issue.

I found that the scrollLeft value of the div around the headers is set back to 0 instead of being set to the same scrollLeft value of the div around the table.  Adding this code resolves the issue, but I think this is actually a bug?  This only affects headers that are positioned absolute.

```js
this.$el.on("reflowed", function(event, $floatContainer) {
    $floatContainer.scrollLeft($floatContainer.siblings("div").scrollLeft());
});
```

**Comment by mkoryak** [comment]
thanks for the report, it sounds like a bug. Im a bit busy this week, I will take a look next week.

**Comment by mkoryak** [comment]
hey, if you can make a jsfiddle, I will be able to fix this faster :)

**Comment by kev22257** [comment]
https://jsfiddle.net/w8hq7kg5/2/

**Comment by kev22257** [comment]
Make sure you scroll to the right on the table before clicking the button.

**Comment by mkoryak** [comment]
thanks, this helps a lot. Right now I am kind of slammed at work, but this will be fixed as soon as I have some down time

**Comment by m3Lith** [comment]
Any news on this?

**Comment by mkoryak** [comment]
sorry, nothing. I have been busy with life/work. Want to submit a PR? 

I think the way to solve this would be to get the scrollLeft of scrollContainer before reflow and set it back to that number after reflow.

**Comment by rshah88** [comment]
I have a potential fix for this bug based on last comment from @mkoryak , only thing different was that scrollLeft had to be set back on floatContainer and not scrollContainer.
I tried 'git push origin' ..but got permission error. How do I push this to origin ? (Sorry, I am 
new with git).

**Comment by mkoryak** [comment]
you need to fork the repo (see where it says "fork" in the top right)
make your changes in the fork. push to the fork and then create a pull request into my repo. you might need to read some docs 🐱

**Comment by rshah88** [comment]
Done...confirm when you can @mkoryak .Thanks.

**Comment by mkoryak** [comment]
@rshah88 thank you! our PR looks good. I merged it and bumped version to 1.4.1
